DQM/L1TMonitorClient/python/L1TObjectsTimingClient_cff.py

```diff
@@ -0,0 +1,303 @@
+       "Ratio_L1TEtSum_BX_MET_Isolated_plus2 'Ratio for L1TEtSum MET isolated bunch for BX=+2' etsum_phi_bx_MET_isolated_plus2 den_etsum_phi_bx_MET_isolated_plus2", 
+       "Ratio_L1TEtSum_BX_METHF_Isolated_minus2 'Ratio for L1TEtSum METHF isolated bunch for BX=-2' etsum_phi_bx_METHF_isolated_minus2 den_etsum_phi_bx_METHF_isolated_minus2",
+       "Ratio_L1TEtSum_BX_METHF_Isolated_minus1 'Ratio for L1TEtSum METHF isolated bunch for BX=-1' etsum_phi_bx_METHF_isolated_minus1 den_etsum_phi_bx_METHF_isolated_minus1",
+       "Ratio_L1TEtSum_BX_METHF_Isolated_0 'Ratio for L1TEtSum METHF isolated bunch for BX=0' etsum_phi_bx_METHF_isolated_0 den_etsum_phi_bx_METHF_isolated_0",
+       "Ratio_L1TEtSum_BX_METHF_Isolated_plus1 'Ratio for L1TEtSum METHF isolated bunch for BX=+1' etsum_phi_bx_METHF_isolated_plus1 den_etsum_phi_bx_METHF_isolated_plus1",
+       "Ratio_L1TEtSum_BX_METHF_Isolated_plus2 'Ratio for L1TEtSum METHF isolated bunch for BX=+2' etsum_phi_bx_METHF_isolated_plus2 den_etsum_phi_bx_METHF_isolated_plus2",
+       "Ratio_L1TEtSum_BX_MHTHF_Isolated_minus2 'Ratio for L1TEtSum MHTHF isolated bunch for BX=-2' etsum_phi_bx_MHTHF_isolated_minus2 den_etsum_phi_bx_MHTHF_isolated_minus2",
+       "Ratio_L1TEtSum_BX_MHTHF_Isolated_minus1 'Ratio for L1TEtSum MHTHF isolated bunch for BX=-1' etsum_phi_bx_MHTHF_isolated_minus1 den_etsum_phi_bx_MHTHF_isolated_minus1",
+       "Ratio_L1TEtSum_BX_MHTHF_Isolated_0 'Ratio for L1TEtSum MHTHF isolated bunch for BX=0' etsum_phi_bx_MHTHF_isolated_0 den_etsum_phi_bx_MHTHF_isolated_0",
+       "Ratio_L1TEtSum_BX_MHTHF_Isolated_plus1 'Ratio for L1TEtSum MHTHF isolated bunch for BX=+1' etsum_phi_bx_MHTHF_isolated_plus1 den_etsum_phi_bx_MHTHF_isolated_plus1",
+       "Ratio_L1TEtSum_BX_MHTHF_Isolated_plus2 'Ratio for L1TEtSum MHTHF isolated bunch for BX=+2' etsum_phi_bx_MHTHF_isolated_plus2 den_etsum_phi_bx_MHTHF_isolated_plus2",
+       "Ratio_L1TEtSum_BX_MHT_Isolated_minus2 'Ratio for L1TEtSum MHT isolated bunch for BX=-2' etsum_phi_bx_MHT_isolated_minus2 den_etsum_phi_bx_MHT_isolated_minus2",
+       "Ratio_L1TEtSum_BX_MHT_Isolated_minus1 'Ratio for L1TEtSum MHT isolated bunch for BX=-1' etsum_phi_bx_MHT_isolated_minus1 den_etsum_phi_bx_MHT_isolated_minus1",
+       "Ratio_L1TEtSum_BX_MHT_Isolated_0 'Ratio for L1TEtSum MHT isolated bunch for BX=0' etsum_phi_bx_MHT_isolated_0 den_etsum_phi_bx_MHT_isolated_0",
+       "Ratio_L1TEtSum_BX_MHT_Isolated_plus1 'Ratio for L1TEtSum MHT isolated bunch for BX=+1' etsum_phi_bx_MHT_isolated_plus1 den_etsum_phi_bx_MHT_isolated_plus1",
+       "Ratio_L1TEtSum_BX_MHT_Isolated_plus2 'Ratio for L1TEtSum MHT isolated bunch for BX=+2' etsum_phi_bx_MHT_isolated_plus2 den_etsum_phi_bx_MHT_isolated_plus2"
+    ),
+    resolution = cms.vstring(),
+    outputFileName = cms.untracked.string("output.root"),
+    verbose = cms.untracked.uint32(0)
+)
+
+# sequences
+l1tObjectsTimingClient = cms.Sequence(
+   l1tObjectsRatioPlots
```

Comment:
Can you try to group the four groups per object into one? They are in different subdirectories but it may be possible to give a histogram title like e.g.
`"Isolated_bunch/Ratio_L1TTau_BX_LastBunch_0 'Ratio for L1TTau last bunch for BX=0' Isolated_bunch/tau_eta_phi_bx_lastbunch_0 Isolated_bunch/den_tau_eta_phi_bx_lastbunch_0"`
It works for sure for the input histograms. For the histogram that is created I am not sure.